Cargo.toml

```diff
@@ -24,10 +24,10 @@ chameleon = "0.1.0"
 scale-info = { version = "1.0.0", features = ["bit-vec"] }
 futures = "0.3.13"
 hex = "0.4.3"
-jsonrpsee-proc-macros = "0.4.1"
-jsonrpsee-ws-client = { version = "0.4.1", default-features = false }
-jsonrpsee-http-client = { version = "0.4.1", default-features = false }
-jsonrpsee-types = "0.4.1"
+jsonrpsee-proc-macros = "0.5.1"
```

Comment:
you could also just use the wrapper crate (jsonrpsee) with `--features ws-client http-client` instead of these imports..

Reply:
Didn't know about that. Thanks!